src/app/pyscada-lite/graphs/doughnut/doughnut.component.ts

```diff
@@ -88,7 +132,7 @@ export class DoughnutComponent implements AfterViewInit, OnChanges {
         ctx.fillStyle = 'black';
         ctx.textAlign = 'center';
         ctx.textBaseline = 'middle';
-        ctx.fillText((100 * value / this.sumValues).toFixed(2) + ' %', left + width / 2, top + height / 2);
+        // ctx.fillText((100 * value / this.sumValues).toFixed(2) + ' %', left + width / 2, top + height / 2);
```

Comment:
Comment is utils ?

Reply:
Handles percentage at center of doughnut, needs real data to adapt